### PR TITLE
Fixed warning in controlsortfiltermodel.cpp

### PR DESCRIPTION
**Pull request by JoergAtGithub**
introduced in #4568

### CODE REVIEW

File: src/control/controlsortfiltermodel.cpp [hunk 1]
```diff
@@ -17,7 +17,7 @@ bool ControlSortFilterModel::sortDescending() const {
 }
 
 void ControlSortFilterModel::sortByColumn(int column, bool descending) {
-    const bool oldColumn = sortColumn();
+    const int oldColumn = sortColumn();
```
Comment (uklotzde) [inline src/control/controlsortfiltermodel.cpp:20]: This is actually a bug that also affects 2.3. Please rebase te branch and retarget the PR.

Comment (JoergAtGithub) [reply src/control/controlsortfiltermodel.cpp:20]: I think this code exist only in Main

Comment (uklotzde) [reply src/control/controlsortfiltermodel.cpp:20]: :see_no_evil: Correct. I checked out the wrong branch.

**Review by uklotzde — APPROVED** [review]
LGTM

**Pull request merged**